src/screens/AuthScreen/AuthScreen.tsx

```diff
@@ -1,32 +1,19 @@
-import React, { useContext, useState } from 'react';
-import {
-  Text,
-  View,
-  TouchableOpacity,
-  Image,
-  ScrollView,
-  ActivityIndicator,
-  SafeAreaView,
-} from 'react-native';
-import WebView from 'react-native-webview';
-import { urls } from '../../constants/appConstant/url';
-import { AuthContext } from '../../context/AuthContext';
-import Images from '../../constants/images/Image';
-import { storeData } from '../../utils/dataStore';
+import React, { useState } from 'react';
+import { Text, View, TouchableOpacity, Image, ScrollView } from 'react-native';
 import Strings from '../../i18n/en';
 import { AuthViewStyle } from './styles';
 import { AuthScreenButton } from './Button';
 import { OtpModal } from './OtpModal';
-import { getUserData } from './Util';
+import { Toast } from 'react-native-toast-message/lib/src/Toast';
 
 const AuthScreen = () => {
-  const { setLoggedInUserData } = useContext(AuthContext);
-  const [githubView, setGithubView] = useState<boolean>(false);
+  // const { setLoggedInUserData } = useContext(AuthContext);
+  // const [githubView, setGithubView] = useState<boolean>(false);
   const [otpCode, setOtpCode] = useState<string>('');
   const [otpModalVisible, setOtpModalVisible] = useState<boolean>(false);
-  const [addressbarURL, setAdressbarURL] = useState<String>('');
-  const [loading, setLoading] = useState(false);
-  const [key, setKey] = useState(1);
+  // const [addressbarURL, setAdressbarURL] = useState<String>('');
```

Comment:
can we remove the comment or add TODO: or a  comment that, why we commented on it?

Reply:
Question: These things were needed when we used to show the webview dressed up to look like a browser. Should I keep these or remove them altogether?

Reply:
add a TODO: will revamp github signIn feature

Reply:
I have added the todo @shreya-mishra